prac_05/word_occurrences.py

```diff
@@ -0,0 +1,22 @@
+"""
+Word Occurrences
+Estimate: 30 min
+Actual:   25 min
+"""
+
```

Comment:
Nice! Looks good 😄 

If you're interested, I did this a very different way (possibly worse).
I got all the unique words using `set(words)` then created a dictionary using those as keys and the `words.count(word)` for the values. 
Your way looks simpler to mine I think.